### PR TITLE
Added SingleChildScrollView/Updated Plugins and Made changes to alert Dialog to aid UX

### PR DESCRIPTION
**Pull request by Itz-kwaz**
I added a singleChildScrollView to make the view scrollable and make views that were previously invisible, visible.
also Removed debug checkedBanner by using the value of isdebugMode.

Removed the hex color plugin and used Color(0xFFFFFFF) instead.
Upgraded web_view_plugin.

### CODE REVIEW

**Comment by FredDominant** [comment]
Hi @Itz-kwaz, thank you for your contribution. Could you look into the conflicts in this request? Looking forward to reviewing this again.

**Comment by Itz-kwaz** [comment]
I'll surely look into the conflicts and resolve them. Thank you very much

File: lib/widgets/bank_account_payment/bank_account_payment.dart [hunk 1]
```diff
@@ -444,15 +444,15 @@ class PayWithBankAccountState extends State<PayWithBankAccount> {
...
       Navigator.of(this._loadingDialogContext).pop();
       this._loadingDialogContext = null;
     }
   }
 
   void _onPaymentComplete(final ChargeResponse chargeResponse) {
-    this._showSnackBar("Transaction complete!");
+    this.showSnackBar("Transaction complete!");
     Navigator.pop(this.context, chargeResponse);
   }
```
Comment (FredDominant) [inline lib/widgets/bank_account_payment/bank_account_payment.dart:458]: Hi @Itz-kwaz, I think it's better to leave these methods private. What do you think?

File: lib/widgets/bank_transfer_payment/bank_transfer_payment.dart [hunk 2]
```diff
@@ -215,7 +215,7 @@ class _PayWithBankTransferState extends State<PayWithBankTransfer> {
     );
   }
 
-  void _closeDialog() {
+  void closeDialog() {
     if (this.loadingDialogContext != null) {
       Navigator.of(this.loadingDialogContext).pop();
       this.loadingDialogContext = null;
```
Comment (FredDominant) [inline lib/widgets/bank_transfer_payment/bank_transfer_payment.dart:221]: Same for the methods in this file

File: pubspec.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 name: "flutterwave"
 description: "Flutterwave official flutter library to accept payment via card, USSD, Mobile Money etc."
-version: 0.0.2
+version: 0.0.1
```
Comment (FredDominant) [inline pubspec.yaml:3]: It's better to leave as `0.0.1`, we'll update this when we publish.

**Review by FredDominant — CHANGES_REQUESTED** [review]
Nice work @Itz-kwaz and thank you. Please check the comments, they basically request you to leave the private methods/fields as is.

**Comment by Itz-kwaz** [comment]
Thank you very much @FredDominant, I'll make all the requested changes. If there is any other way I could contribute, please do let me know I'll be more than glad to do so.

**Review by FredDominant — APPROVED** [review]

**Pull request merged**